run_gencall.sh

```diff
@@ -0,0 +1,45 @@
+# Download cluster/probe metadata
+aws s3 cp s3://genotype-datasets/marker_files/v6_chip_production_probe_lists/Embark_2021_260k_20063270_A1_20211122.egt .
+aws s3 cp s3://genotype-datasets/marker_files/v6_chip_production_probe_lists/Embark_2021_260k_20063270_A1.bpm .
+aws s3 cp s3://genotype-datasets/marker_files/v6_chip_production_probe_lists/Embark_2021_260k_20063270_A1.csv .
+aws s3 sync s3://illumina-embark-sample-level-deliveries/2021-12-14/31001805234805/IDATs/205881370069/ idats
+
+# Copy the ILMN data to the test bucket for safekeeping
+aws s3 sync s3://illumina-embark-sample-level-deliveries/2021-12-14/31001805234805 s3://scratch-embark/iaap-test/
+
+# Download an example idat from a new dog
+iaap-cli gencall Embark_2021_260k_20063270_A1.bpm Embark_2021_260k_20063270_A1_20211122.egt outputs -f idats -g
+aws s3 sync outputs s3://scratch-embark/iaap-test/
+# Grab canfam 4 as reference data
```

Comment:
Why cf4?

Reply:
Needed to ask about SRY first, but got details on that this morning

Reply:
The 3.1_and_SRY assembly has some issues that caused things to break (naming conventions in particular), but now that I know more about how it was created I can work around those issues